common/types/config.ts

```diff
@@ -11,6 +11,7 @@ export const configSchema = schema.object({
     enabled: schema.boolean({ defaultValue: false }),
     trace: schema.boolean({ defaultValue: true }),
     feedback: schema.boolean({ defaultValue: true }),
+    regenerateButton: schema.boolean({ defaultValue: true }),
```

Comment:
I suggest rename it to `regenerateMessage` to avoid mentioning `Button` in the flag, because the UI might be different in the future.